tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/JsonWriter.java

```diff
@@ -0,0 +1,45 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import com.google.gson.Gson;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Utility for converting objects into JSON format and writing them into a file.
+ */
+public class JsonWriter {
+    /**
+     * Convert and save object into outputFile as JSON.
+     *
+     * @param obj        to be converted into JSON format.
+     * @param outputFile destination where the converted result is to be saved
+     */
+    public static void writeObjectIntoJsonFormat(Object obj, Path outputFile) {
+        String objectInJson = new Gson().toJson(obj);
+        try {
+            writeOutputFile(objectInJson, outputFile);
+        } catch (IOException ex) {
+            Logger.getLogger(JsonWriter.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
Bad idea to not rethrow exceptions in a generic helper like this. I also don't think it's necessary to log the exception here - let whatever handles the final exception do logging.